src/UI/EmailHTMLTooltip.js

```diff
@@ -37,11 +37,19 @@ ${this.options.css}
                 this.addressEl.textContent = formatDuckAddress(addresses.personalAddress)
             }
         }
+
+        const personalAddressUsed = this.device.personalAddressUsed.bind(this.device)
+        const privateAddressUsed = this.device.privateAddressUsed.bind(this.device)
+
         this.registerClickableButton(this.usePersonalButton, () => {
             this.fillForm('personalAddress')
+            personalAddressUsed()
+            console.log('personalAddress selected')
         })
         this.registerClickableButton(this.usePrivateButton, () => {
             this.fillForm('privateAddress')
+            privateAddressUsed()
```

Comment:
This will then read something like `this.device.firePixel('privateAddressUsed')`.